tests/integration-tests-topologies/src/main/java/org/apache/pulsar/tests/containers/BrokerContainer.java

```diff
@@ -29,4 +31,29 @@ public BrokerContainer(String clusterName, String hostName) {
         super(
             clusterName, hostName, hostName, "bin/run-broker.sh", BROKER_PORT, INVALID_PORT);
     }
+
+    public void stopBroker() {
```

Comment:
I am not sure these two methods are needed.

 you should configure broker configuration by setting envs before starting the docker container. comments as below.